docs/pages/build/introduction.md

```diff
@@ -2,4 +2,8 @@
 title: Introduction
 ---
 
-This is why this exists and how it is different from the previous thing.
\ No newline at end of file
+**EAS Builds** is a service for building app binaries for distribution on the Apple App Store and Google Play Store. You can take advantage of it for both managed and bare Expo projects. This service is going to replace the existing build service entirely.
```

Comment:
```suggestion
**EAS Builds** is a service for building app binaries for testing and distribution on the Apple App Store and Google Play Store. You can take advantage of it for both managed and bare Expo projects. This service is going to replace the existing build service entirely.
```

Since you can install an apk directly onto your device, and (I'm guessing) provide an ad-hoc provisioning profile for the iOS build, no reason to only specify distribution via the stores